interface/therapy_groups/therapy_groups_views/listGroups.php

```diff
@@ -0,0 +1,399 @@
+                    <p class="failed_message"><?php echo xlt($deletion_response['message']);?></p>
+                </div>
+            </div>
+        </div>
+    <?php endif ?>
+
+    <!---------- FILTERS SECTION ------------->
+    <a id="show_filters" class="btn btn-alert"><?php echo xlt("Show Filters")?></a>
+    <a id="hide_filters" class="btn btn-alert" style="display: none;"><?php echo xlt("Hide Filters")?></a>
+    <button id="clear_filters" class="btn"><?php echo xlt("Clear Filters")?></button>
+    </br></br>
+    <div id="filters" style="display: none;">
+        <div class="row">
+            <div class=" form-group col-md-2">
+                <label class="" for="group_name_filter"><?php echo xlt('Group Name');?>:</label>
+                <input type="text" class="form-control" id="group_name_filter" placeholder="" >
+            </div>
+            <div class=" form-group col-md-2">
+                <label class="" for="group_id_filter"><?php echo xlt('Group Id');?>:</label>
+                <input type="number" class="form-control" id="group_id_filter" placeholder="" >
+            </div>
+            <div class=" form-group col-md-2">
+                <label class="" for="group_type_filter"><?php echo xlt('Group Type');?>:</label>
+                <select type="text" class="form-control" id="group_type_filter" placeholder="" >
+                    <option value=""><?php echo xlt('choose');?></option>
+                    <?php foreach ($group_types as $type):?>
+                        <option value="<?php echo attr($type);?>"><?php echo xlt($type) ;?></option>
+                    <?php endforeach; ?>
+                </select>
+            </div>
+            <div class=" form-group col-md-2">
+                <label class="" for="group_status_filter"><?php echo xlt('Status');?>:</label>
+                <select type="text" class="form-control" id="group_status_filter" placeholder="" >
+                    <option value="<?php echo attr($statuses[10]); ?>"><?php echo xlt($statuses[10]);?></option>
+                    <?php foreach ($statuses as $status):?>
+                        <?php if($status != $statuses[10]): ?>
+                            <option value="<?php echo attr($status);?>"><?php echo xlt($status) ;?></option>
+                        <?php endif; ?>
+                    <?php endforeach; ?>
+                    <option value="all"><?php echo xlt("all");?></option>
+                </select>
+            </div>
+            <div class=" form-group col-md-2">
+                <label class="" for="counselors_filter"><?php echo xlt('Main Counselors');?>:</label>
+                <select type="text" class="form-control" id="counselors_filter" placeholder="" >
+                    <option value=""><?php echo xlt('choose');?></option>
+                    <?php foreach ($counselors as $counselor):?>
+                        <option value="<?php echo attr($counselor);?>"><?php echo xlt($counselor) ;?></option>
```

Comment:
Is the $counselor names. If so, should not translate, and instead should use text() function.